docs/advanced_reference/commands/autonomy_build-image.md

```diff
@@ -2,11 +2,19 @@ Builds the agent image for a service.
 
 The command must be executed within the service directory. That is, a directory containing the service configuration file (`service.yaml`). Labels for images generated using this command will follow the format `valory/oar-<PUBLIC_ID>`.
 
-To build an image for a specific agent without fetching a service, run
+To build an image for a specific agent without fetching a service navigate to the service directory, and run
```

Comment:
How can the user navigate to a service repository without fetching the service?

Reply:
My bad =